bpf/lib/conntrack.h

```diff
@@ -451,7 +451,7 @@ ipv6_extract_tuple(struct __ctx_buff *ctx, struct ipv6_ct_tuple *tuple)
 	if (ret < 0)
 		return DROP_CT_INVALID_HDR;
 
-	return CTX_ACT_OK;
+	return 0;
```

Comment:
Could you please fix up the corresponding usage by `IP_TUPLE_EXTRACT_FN()`  in `bpf/lib/proxy.h` ?